site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -3,3 +3,61 @@ api: runtime
 ---
 
 <!-- Intentionally blank -->
+
+## Examples
+
+Using a content script to inject an extension asset on a page.
+
+```js
+//// content.js
+{ // Block used to avoid setting global variables
+  let el = document.createElement('img');
+  el.src = chrome.runtime.getURL('logo.png');
+  document.body.append(el);
+}
+```
+
+A basic example of content script injected into a page and background script passing messages to
+each other. This exchange is modeled on a [TCP handshake][handshake].
+
+```js
+//// content.js
+// 1. Send the background a message requesting receipt confirmation.
+console.log('[content] sending content-syn');
+chrome.runtime.sendMessage('content-syn');
+
+chrome.runtime.onMessage.addListener((message, sender) => {
+  if (message === 'bg-syn-ack') {
+    // 3. BG syn-ack received, acknowledged receipt.
+    console.log('[content] received bg-syn-ack');
+    console.log('[content] sending content-ack');
+    chrome.runtime.sendMessage('content-ack');
+  }
+});
+
```

Comment:
break into two separate blocks (I think @awfuchs had this comment)

Reply:
Done.